src/importer/users-importer.js

```diff
@@ -0,0 +1,59 @@
+var Promise = require('bluebird');
```

Comment:
'Promise' is defined in a future version of JavaScript. Use a different variable name to avoid migration issues.<br>'require' is not defined.